Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.9.5-slim-buster as base
+
+RUN pip install flask
```

Comment:
No, you don't want to install dependencies with pip. `poetry install` will handle installing Flask (and will specify the correct version)